src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -378,6 +420,78 @@ public void removeCitizen(final ICitizenData citizen)
         super.removeCitizen(citizen);
     }
 
+    /**
+     * Bonemeal fungi in netherTrees
+     */
+
+    private void bonemealFungi()
```

Comment:
I think the javadoc has to be more precise to what this method does (also move it down by one).

Besides that, I think we should grow at most 1 tree per tick here (such that we don't grow all at the same time)

Reply:
so, on grow, probably do a return;